### ISSUE TITLE
[BUG] UniGetUI 3.1.0 installation is stuck

### ISSUE THREAD

**Issue by skycommand** [marticliment/UniGetUI#2473]
### Please confirm these before moving forward

- [X] I have searched for my issue and have not found a work-in-progress/duplicate/resolved issue.
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
- [X] I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section for solutions.
- [X] This issue is about a bug (if it is not, please use the [correct template](https://github.com/marticliment/WingetUI/issues/new/choose)).

### UniGetUI Version

3.1.0

### Windows version, edition, and architecture

Windows 10 v22H2 build 19045.4651 (July 2024 update)

### Describe your issue

Today, I removed WinGetUI and installed UniGetUI 3.1.0.

UniGetUI's installer remained stuck at the last stage, which installs `Microsoft.WinGet.Client`.

After waiting ten minutes, I got tired and decided to see what the UniGetUI installer was doing. Process Informer revealed that the UniGetUI installer had executed the 32-bit Windows PowerShell from `C:\Windows\SysWOW64\WindowsPowerShell\v1.0\powershell.exe`. This process was stuck for 9 minutes and 56 seconds.

![Screenshot 1](https://github.com/marticliment/UniGetUI/assets/17097175/a386ff19-32ff-45cd-9c68-ebc828b43201)

I looked at the process's list of windows. Indeed, PowerShell had a hidden window.

![Screenshot 2](https://github.com/marticliment/UniGetUI/assets/17097175/5a828498-434f-4eae-9ef8-aeb7a48ee58a)

I ordered Process Informer to maximize the hidden window. This is what I got.

![Screenshot 3](https://github.com/marticliment/UniGetUI/assets/17097175/c1b4e0c9-8d35-43a4-84cd-ca3f60791301)

As you can see, PowerShell is trying to install a module called `Microsoft.WinGet.Client` but is now asking me to consent to the installation of a NuGet provider. It is waiting for me to say "Yes", "No", "Suspend", or "Help".

**Please pay careful attention to this part:** I did not consent! I selected "No". The installation ended. UniGetUI started, asked me for permission to install `Microsoft.WinGet.Client`, and upon my confirmation installed it without a problem using the 64-bit PowerShell.

### Steps to reproduce the issue

See above.

### UniGetUI Log

```text
This is an installer issue; no logs are available.
```

### Package Managers Logs

```text
This is an installer issue; no logs are available.
```

**Comment by marticliment** [comment]
Will investigate

**Comment by marticliment** [comment]
Fixed by a6a6900b30f3f9e7674ae1b2c86179b70481a3d7
